src/Reports/SiteSummary.php

```diff
@@ -84,6 +88,28 @@ public function getReportField()
         return $grid;
     }
 
+    public function getCMSFields()
+    {
+        $fields = parent::getCMSFields();
+        $alerts = $this->getAlerts();
+        if ($alerts) {
+            $summaryInfo = '<div class="site-summary message warning">' . implode("\n", $alerts) . '</div>';
+            $fields->unshift(LiteralField::create('AlertSummary', $summaryInfo));
+        }
+        return $fields;
+    }
+
+    /**
+     * Return a list of alerts to display in a message box above the report
+     * A combination of free text fields - combined alerts as opposed to a message box per alert.
```

Comment:
Plz add a return phpdoc

Reply:
This doesn't auto-resolve because the line it's on hasn't changed.
But there is now a return php doc 2 lines down (after an empty one) in this file.